napari/utils/events/event.py

```diff
@@ -518,8 +518,9 @@ def __call__(self, *args, **kwargs):
             for cb in rem:
                 self.disconnect(cb)
         finally:
-            if event._pop_source() != self.source:
-                raise RuntimeError("Event source-stack mismatch.")
+            pass
+            # if event._pop_source() != self.source:
+            #     raise RuntimeError("Event source-stack mismatch.")
```

Comment:
Question here - what should we do about this?

Reply:
Can you remind me the context here?  And why this particular line is problematic for you in this PR?

Reply:
without the comment I get a truthy comparison error, I think it's a comparison with the dataclass itself (i.e. not a numpy array or something). I can uncomment out and print error message when I get the chance.

I'm not quite sure why we need that check, and the comment doesn't necessarily bother me, but is a bit funny. Maybe getting the equality check working for dataclass fixes this, or it could me something else and I'm wrong. Will look in more

Reply:
hmm looks like this might be ok ... I can't seem to reproduce an error with this not commented, so maybe it can stay. could have been something causing problems when i was implementing that is fixed now ....